variables.tf

```diff
@@ -54,6 +54,12 @@ variable "runners_name" {
   type        = "string"
 }
 
+variable "runners_executor" {
```

Comment:
Can you add an dedicated example for newly added executor, looks like you can simply base the example on the default and then just enable the docker

Reply:
FIxed